<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>681</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startCount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Пуск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopCount">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="countParam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры счёта</string>
    </property>
    <widget class="QLineEdit" name="stepfd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="startpointlb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Начальные условия:</string>
     </property>
    </widget>
    <widget class="QLabel" name="steplb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Начальный шаг:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="u0fd">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="x0fd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="endlb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Конечная точка</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="endfd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="isInfch">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>inf</string>
     </property>
    </widget>
    <widget class="QLabel" name="epslb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ε погр, ε гр.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="epsCtrlfd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="maxSteplb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Макс. шагов.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="epsBrdfd">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxStepfd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="info">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <widget class="QLabel" name="tasklb">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Задача: </string>
     </property>
    </widget>
    <widget class="QLabel" name="stepUplb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Увел. шага</string>
     </property>
    </widget>
    <widget class="QLabel" name="stepDwnlb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Уменьш. шага:</string>
     </property>
    </widget>
    <widget class="QLabel" name="stepUp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="stepDwn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="task">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="taskParam">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры задачи</string>
    </property>
    <widget class="QLabel" name="a1lb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>a1</string>
     </property>
    </widget>
    <widget class="QLabel" name="a3lb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>a3</string>
     </property>
    </widget>
    <widget class="QLabel" name="mlb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="a1fd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="a3fd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mfd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseCount">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Пауза</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>571</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>x0fd</tabstop>
  <tabstop>u0fd</tabstop>
  <tabstop>stepfd</tabstop>
  <tabstop>endfd</tabstop>
  <tabstop>isInfch</tabstop>
  <tabstop>epsCtrlfd</tabstop>
  <tabstop>epsBrdfd</tabstop>
  <tabstop>maxStepfd</tabstop>
  <tabstop>a1fd</tabstop>
  <tabstop>a3fd</tabstop>
  <tabstop>mfd</tabstop>
  <tabstop>startCount</tabstop>
  <tabstop>pauseCount</tabstop>
  <tabstop>stopCount</tabstop>
  <tabstop>clear</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
